sas9.4-viya/azuredeploy.json

```diff
@@ -0,0 +1,3683 @@
+            } 
+        }
+    },
+    "variables": {
+        "_artifactsLocationSasToken": "[parameters('_artifactsLocationSasToken')]",
+        "access_policy_template": "[concat(parameters('_artifactsLocation'),'nestedtemplates/access_policy.json', variables('_artifactsLocationSasToken'))]",
+        "ansible_nw_interface": "[concat(parameters('SAS Application Name'),'_',variables('ansible_vm_name'),'_nic_',variables('resourceGroupUniqueString'))]",
+        "ansible_nw_sg": "[concat(parameters('SAS Application Name'),'_',variables('ansible_vm_name'),'_nsg_',variables('resourceGroupUniqueString'))]",
+        "ansible_pub_nw_interface": "[concat(parameters('SAS Application Name'),'_',variables('ansible_vm_name'),'_pub_nic_',variables('resourceGroupUniqueString'))]",
+        "ansible_ssl_copy_sh": "[concat(parameters('_artifactsLocation'),'scripts/ansible_ssl.sh', variables('_artifactsLocationSasToken'))]",
+        "ansible_startup_script_sh": "[concat(parameters('_artifactsLocation'),'scripts/ansible_setup.sh', variables('_artifactsLocationSasToken'))]",
+        "ansible_vm_name": "ansible",
+        "cascontroller_nw_interface": "[concat(parameters('SAS Application Name'),'_',variables('cascontroller_vm_name'),'_nic_',variables('resourceGroupUniqueString'))]",
+        "cascontroller_nw_sg": "[concat(parameters('SAS Application Name'),'_',variables('cascontroller_vm_name'),'_nsg_',variables('resourceGroupUniqueString'))]",
+        "cascontroller_vm_name": "cascontroller",
+        "casworker_nw_sg": "[concat(parameters('SAS Application Name'),'_',variables('casworker_vm_name'),'_nsg_',variables('resourceGroupUniqueString'))]",
+        "casworker_vm_name": "casworker",
+        "certificatesPermissions": [
+            "import",
+            "get",
+            "list"
+        ],
+        "compute_config_sh": "[concat(parameters('_artifactsLocation'),'scripts/compute_config.sh', variables('_artifactsLocationSasToken'))]",
+        "compute_install_sh": "[concat(parameters('_artifactsLocation'),'scripts/compute_install.sh', variables('_artifactsLocationSasToken'))]",
+        "compute_nw_interface": "[concat(parameters('SAS Application Name'),'_',variables('compute_vm_name'),'_nic_',variables('resourceGroupUniqueString'))]",
+        "compute_nw_sg": "[concat(parameters('SAS Application Name'),'_',variables('compute_vm_name'),'_nsg_',variables('resourceGroupUniqueString'))]",
+        "compute_vm_name": "compute",
+        "custom_data_cas": "#cloud-config\n mounts:\n   - [ ephemeral0, /cascache]",
+        "custom_data_sas": "#cloud-config\n mounts:\n   - [ ephemeral0, /saswork]",
+        "diagnostic_storagegroup_name": "[toLower(concat(parameters('SAS Application Name'),'diag', variables('resourceGroupUniqueString')))]",
+        "domain_name": "internal.cloudapp.net",
+        "enabledForDeployment": true,
+        "enabledForDiskEncryption": false,
+        "enabledForTemplateDeployment": true,

```

Comment:
why is this needed?

Reply:
removed and hardcoded in the properties of the resource itself.